Kernel/VM/Space.cpp

```diff
@@ -158,41 +158,45 @@ Region* Space::find_region_containing(const Range& range)
     return nullptr;
 }
 
+/**
+* @brief returns all whole regions in range and only when all matches are full matches
```

Comment:
I've removed this helper, because I didn't need it anymore...

And I've renamed the second more general helper to something simmilar to your suggestion